doc/qbk/07_concepts/Fields.qbk

```diff
@@ -153,6 +161,9 @@ In this table:
             empty, the Transfer-Encoding field shall not appear when
             the associated __FieldsReader__ serializes the fields.
         ]]
+
+        If the result of adjusting the field value produces an empty
```

Comment:
This reads weird. Prefer:

>If adjusting the field value results in an empty string, the field is removed from the container.

Or

>If the adjusted field value is empty, the field is removed from the container.

Reply:
Actually I wrote it that way on purpose. There is no requirement that implementations of **Fields** store strings at all. It should actually read:

*If the resulting adjusted field value would produce an empty string when serialized, the container behaves as if the field is removed*